<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoachWindow</class>
 <widget class="QMainWindow" name="CoachWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #161A22;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>1851</width>
      <height>971</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22252D;
border-radius:40px;
border: 1px solid #FFFFFF; /* Subtle blue-gray border */
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>85</width>
      <height>85</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/logo football.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>310</y>
      <width>15</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#line_2 {
    background-color: #D8A353; /* Match the exact background color */
    border: 2px solid #2D3748; /* Darker border to contrast with background */
    border-radius: 3px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>314</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/coach.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="coachButton">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>327</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:white;
border:0px;</string>
    </property>
    <property name="text">
     <string>Coach</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>790</y>
      <width>135</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #393C44;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>850</y>
      <width>28</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/language.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>920</y>
      <width>28</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/img/logout.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="LogOutButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>920</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:#393C44;
border:0px;</string>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="LogOutButton_2">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>850</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:#393C44;
border:0px;</string>
    </property>
    <property name="text">
     <string>About</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>1672</width>
      <height>515</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/img/HEADER.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>530</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Tasks :</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>580</y>
      <width>701</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListView#listView {
    background-color: #2D3748; /* Matches the sidebar background */
    border: 1px solid #4A4E58; /* Subtle border matching chart borders */
    border-radius: 5px; /* Slightly rounded corners for a modern look */
    color: #FFFFFF; /* White text for contrast */
    font: 12pt &quot;Arial&quot;; /* Matches the font style used elsewhere */
}

QListView#listView::item {
    background-color: #2D3748; /* Item background matches the list background */
    padding: 10px; /* Spacing for better readability */
    color: #FFFFFF; /* White text for items */
}

QListView#listView::item:hover {
    background-color: #F5A623; /* Golden highlight on hover to match accents */
    color: #1A2526; /* Dark text for contrast on hover */
}

QListView#listView::item:selected {
    background-color: #D8A353; /* Slightly lighter gold for selected items */
    color: #1A2526; /* Dark text for contrast when selected */
    border: none; /* Remove any default border */
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>590</y>
      <width>389</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/img/viewLabel.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Viewplayers">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>620</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Style for the pushButton_4 */
QPushButton#Viewplayers {
    background-color: #F5A623; /* Original background color */
    color: #1A2633; /* Dark blue text */
    border: 1px solid white; /* Subtle white border */
    border-radius: 20px;
    min-width: 62px;
    min-height: 62px;
    
}

QPushButton#Viewplayers:hover {
    background-color: #ffdba2; /* Lighter shade on hover */
    min-width: 70px; /* Slight size increase */
    min-height: 70px;
    
}

QPushButton#Viewplayers:pressed {
    background-color: #d68f1e; /* Darker shade when pressed */
    min-width: 62px; /* Original size */
    min-height: 62px;
   
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/img/view.png</normaloff>:/img/view.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>42</width>
      <height>42</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>710</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Browse the complete list of registered players.
View details</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>580</y>
      <width>401</width>
      <height>420</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/img/matches.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>620</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>MATCHES</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>690</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;View matches and set player &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;lineups.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MatchesButton">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>850</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 1px solid #4A4E58; /* Subtle border matching chart borders */
    border-radius: 15px; /* Slightly rounded corners for a modern look */
background-color: #BEF8B6;
color:black;</string>
    </property>
    <property name="text">
     <string>                   HERE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>858</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/img/here.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>1870</x>
      <y>50</y>
      <width>16</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/img/Vectorrrr.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>910</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: linear-gradient(45deg, #0288D1 0%, #03A9F4 100%);
border-radius: 30px;
transition: background 0.3s, transform 0.2s;
}
QPushButton:hover {
 background: linear-gradient(45deg, #03A9F4 0%, #4FC3F7 100%);
 transform: scale(1.1);
}
QPushButton:pressed {
 background: linear-gradient(45deg, #01579B 0%, #0288D1 100%);
 transform: scale(0.9);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/logout.png</normaloff>:/images/logout.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="profileButton">
    <property name="geometry">
     <rect>
      <x>1790</x>
      <y>20</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: linear-gradient(45deg, #0288D1 0%, #03A9F4 100%);
border-radius: 30px;
transition: background 0.3s, transform 0.2s;
}
QPushButton:hover {
 background: linear-gradient(45deg, #03A9F4 0%, #4FC3F7 100%);
 transform: scale(1.1);
}
QPushButton:pressed {
 background: linear-gradient(45deg, #01579B 0%, #0288D1 100%);
 transform: scale(0.9);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/logout.png</normaloff>:/images/logout.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_8">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:#393C44;</string>
    </property>
    <property name="text">
     <string>Role</string>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_9">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:#393C44;</string>
    </property>
    <property name="text">
     <string>User name</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentUserPhotoLabel">
    <property name="geometry">
     <rect>
      <x>1790</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1E1F2D;
border: 1px solid #3A3D4A;
border-radius: 	30px;
box-shadow: 0 4px 10px rgba(0, 0, 0, 0.2);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
